.github/workflows/ci.yml

```diff
@@ -496,8 +511,9 @@ jobs:
       matrix:
         target:
           - name: x86_64-unknown-haiku
+            exclude_features: "taskdump" # taskdump is only available on Linux
           - name: armv7-sony-vita-newlibeabihf
-            exclude_features: "process,signal,rt-process-signal,full"
+            exclude_features: "process,signal,rt-process-signal,full,taskdump"
```

Comment:
This changed the matrix, and also changed the name of this job, we need to change the repo setting to unblock the CI.

See also <https://github.com/tokio-rs/tokio/pull/7655/files#r2393408483>.